### ISSUE TITLE
Notebook active cell is not visible when notebook is large (using keyboard nav)

### ISSUE THREAD

**Issue by VasuBhog** [microsoft/azuredatastudio#18271]
Version: 1.35.0-insider
Commit: fdf7bbcf607771a6ea52309f5895b6cb0903153f
Date: 2022-02-09T19:43:21.778Z (2 hrs ago)
VS Code: 1.59.0
Electron: 13.5.0
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.2.0

Steps to Reproduce:

Seems to only happen with code cells.

1. Have a Notebook that is large enough that has cells go past the bottom of the viewable area (notebook has a vertical scrollbar)
2. Use the keyboard down arrow (pressing not holding down) to go all the way to the bottom and see that the active cell on the last cell
3. Press the down arrow past the last cell and see it disappear back to the top cell.

Expected: Active cell should be shown and the view should scroll back to the top.
Current: The view does not change and I need to use the scrollbar to find the active cell.

**Comment by barbaravaldez** [comment]
Duplicate of https://github.com/microsoft/azuredatastudio/issues/18270
